tests/test_wikipedia_linking.py

```
@@ -0,0 +1,258 @@
+
+# Integration tests (require actual API keys and network access)
+@pytest.mark.integration
+class TestWikipediaLinkingIntegration:
+    """Integration tests for Wikipedia linking (require API access)"""
+    
+    @pytest.mark.skip(reason="Requires OpenAI API key")
+    def test_real_entity_linking(self):
+        """Test actual entity linking with real APIs"""
+        service = WikipediaLinkingService(similarity_threshold=0.3)
+        
+        result = service.link_entity_to_wikipedia(
```

Pylint (3.3.7)

[error] Instance of 'WikipediaLinkingService' has no 'link_entity_to_wikipedia' member

(E1101)